README.md

```diff
@@ -1,2 +1,49 @@
-# hellomicroverse
-set up hello microverse project
+📖 About the Project
+🛠 Built With
+Setup
+Prerequisites
+Install
+Usage
+Run tests
+Deployment
+👥 Authors
+📝 License
+📖 [HELLOMICROVERSE]
+Describe your project in 1 or 2 sentences.
+
+[HELLOMICROVERSE] is a simple html page with text hello microverse with background color.
```

Comment:
Great job adding your readme file to your project👍. However, please stick to the structure of the microverse readme [template](https://github.com/microverseinc/readme-template)🙏. This means copying the template into your repository and only filling out each section, as opposed to editing the structure. The microverse readme has been created as a great template that includes all the necessary sections for documenting our projects✨. Please make sure to stick to the structure and fill out all sections with information relating to your project.